docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1737,6 +1735,182 @@ As a response to blocked request AdGuard returns a short video placeholder.
 >
 > Rules with `$mp4` modifier are not supported by AdGuard Content Blocker, AdGuard for iOS and Safari.
 
+### Rule priorities {#rule-priorities}
+Each rule has its own priority, which is necessary when several rules match the request and the filtering system needs to select one of them.
+Priority is measured by a natural integer. The higher the priority of a rule, the higher its priority value.
```

Comment:
"natural integer" is redundant :) Moreover, "natural" may or may not include zero depending on who you ask, so "positive integer" is probably a better wording.

Reply:
@ngorskikh hmm, I asked google and it said that a natural number means not including zero, as I learned in school. where can a natural number include zero?

I'm asking out of interest, not for the sake of argument.

I will replace it with a positive integer in the documentation as being more easily understandable.